Scenes/MainMenuUI/MainMenuUI.tscn

```diff
@@ -90,7 +90,6 @@ theme_override_font_sizes/font_size = 20
 text = "Loading..."
 
 [node name="CreateServerBtn" type="Button" parent="MarginContainer/VBoxContainer/VBoxContainer"]
-visible = false
```

Comment:
Ideally, this would be toggled at runtime, based on whether `debug` mode is active.